checks.d/mongo.py

```diff
@@ -829,7 +829,7 @@ def total_seconds(td):
             pass
 
         dbnames = cli.database_names()
-        GAUGE(self, 'mongodb.dbs', len(dbnames), tags=tags)
+        self.gauge('mongodb.dbs', len(dbnames), tags=tags)
```

Comment:
One question, I see the tags list is extended in the big `try...except` block above. I'm not sure if all of those `tags` would apply to this particular metric. My knowledge of the mongo check is a bit limited, we should make sure of that before merging.